operations/tempo-mixin/alerts.libsonnet

```diff
@@ -84,17 +84,33 @@
           },
           {
             // wait 5m for failed flushes to self-heal using retries
-            alert: 'TempoIngesterFlushesFailing',
+            alert: 'TempoIngesterFlushesUnhealthy',
             expr: |||
-              sum by (%s) (increase(tempo_ingester_failed_flushes_total{}[1h])) > %s and
-              sum by (%s) (increase(tempo_ingester_failed_flushes_total{}[5m])) > 0
+              sum by (%s) (increase(tempo_ingester_flush_failed_retries_total{}[1h])) > %s and
```

Comment:
what do we think about keying the "unhealthy" warning alert  on the old metric and the "failing" critical alert on the retries metric?

then for consistency keep the `for: 5m` on both?

Reply:
That could be reasonable.  The goal being to always know when a flush failed at a warning level, but only get paged when a retry fails for the `5m` duration?

Reply:
Yeah, that's what i was thinking. :+1:

Reply:
Okay, I've made the update.